src/System.Security.Cryptography.Primitives/tests/CryptoStream.cs

```diff
@@ -158,6 +158,20 @@ public static void NestedCryptoStreams()
             }
         }
 
+#if netcoreapp11 
+        [Fact]
+        public static void Clear()
+        {
+            ICryptoTransform encryptor = new IdentityTransform(1, 1, true);
+            using (MemoryStream output = new MemoryStream())            
+            using (CryptoStream encryptStream = new CryptoStream(output, encryptor, CryptoStreamMode.Write))
+            {
+                encryptStream.Clear();
+                Assert.Throws<NotSupportedException>(() => encryptStream.Write(new byte[] { 1, 2, 3, 4, 5 }, 0, 5));
```

Comment:
I totally would not have expected this behavior, but I see that it's what desktop would have done.  Good test, good test.